src/core/client/http-client.js

```diff
@@ -71,7 +71,12 @@ class HttpClient {
     }
 
     return requestPromise
-      .then(response => response.data);
+      .then(response => response.data)
+      .catch(err => {
+        if (err.code === 429) {
```

Comment:
Change the status code in the test to see how you can access the status code:
https://github.com/logzio/slack-integration/blob/master/src/core/client/http-client.spec.js#L72

In case the status code is not 429 rethrow the exception

Reply:
And please add a test